pkg/anonymiser/anonymiser.go

```diff
@@ -17,37 +17,29 @@ const literalPrefix = "literal:"
 // anonymiser anonymises MySQL tables
 type anonymiser struct {
 	reader.Reader
+	tables config.Tables
 }
 
 // NewAnonymiser returns an initialised instance of MySQLAnonymiser
-func NewAnonymiser(source reader.Reader) reader.Reader {
-	return &anonymiser{source}
+func NewAnonymiser(source reader.Reader, tables config.Tables) reader.Reader {
+	return &anonymiser{source, tables}
 }
 
-func (a *anonymiser) ReadTable(table string, rowChan chan<- *database.Row) error {
-	logger := log.WithField("table", table)
+func (a *anonymiser) ReadTable(tableName string, rowChan chan<- *database.Row) error {
+	logger := log.WithField("table", tableName)
 	logger.Info("Loading anonymiser config")
-	// Find all columns that need to be anonimized
-	columnFakers, err := a.fetchColumnToAnonimise(table)
+
+	table, err := a.tables.FindByName(tableName)
 	if err != nil {
```

Comment:
Based on `FindByName` we now MUST configure all tables. I think this maybe a little bit of a based User experience

Reply:
True, maybe we should simply log that the table is not configured.